### ISSUE TITLE
Does text exposed in title attributes fail SC 1.4.4 Resize Text

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#2727]
While use of title attributes has many challenges with other WCAG success criteria - it would be good to have consensus on whether use of a title attribute would fail SC 1.4.4 resize text because the implementation by the browser is not resizable and is controlled by the user agent.

**Comment by patrickhlauke** [comment]
I would say no, because it's down to the browser - the same way that we wouldn't ding a site for how the user can't resize a file open dialog or `alert()` dialog or similar.

**Comment by bruce-usab** [comment]
I agree that <q>controlled by the the user agent</q> gets a pass elsewhere, and that logic can be enough.

Regardless, is it canonical that `title` attribute be visually presented?  AFAIK the only formal semantic is as a component of Accessible Name computation.

**Comment by JAWS-test** [comment]
I think this should be clarified in general:
* There are some SCs in WCAG with the exception that if something is from the user agent and has not been changed, it does not apply (e.g 1.4.11, 1.4.13, 2.4.11).
* For all other SCs, I assume that the exception does not apply. Of course the browsers are responsible, but if the browsers don't do it, the thing is considered not [Accessibility supported](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported) and must not be used
